src/server/active_tran_server.cpp

```diff
@@ -93,17 +87,34 @@ active_tran_server::compute_consensus_lsa ()
    */
   std::sort (collected_saved_lsa.begin (), collected_saved_lsa.end ());
 
-  const auto consensus_lsa = collected_saved_lsa[cur_node_cnt - quorum];
+  LOG_LSA consensus_lsa;
+
+  if (cur_node_cnt < quorum)
+    {
+      consensus_lsa = NULL_LSA; // the quorum is not met.
+    }
+  else
+    {
+      consensus_lsa = collected_saved_lsa[cur_node_cnt - quorum];
+    }
```

Comment:
Nit.
When the quorum is not met (a rather exception situation), the only extra useless operation is the `std::sort`.
So:
```suggestion
    LOG_LSA consensus_lsa;
    if (cur_node_cnt < quorum)
    {
      if (prm_get_bool_value (PRM_ID_ER_LOG_QUORUM_CONSENSUS))
        {
          // if the quorum is not met, only sort if we're about to log the details (see below); otherwise the sorting is useless   
          std::sort (collected_saved_lsa.begin (), collected_saved_lsa.end ());
        }
      consensus_lsa = NULL_LSA; // the quorum is not met.
    }
  else
    {
      // always do the sort if the quorum is met:
      std::sort (collected_saved_lsa.begin (), collected_saved_lsa.end ());
      consensus_lsa = collected_saved_lsa[cur_node_cnt - quorum];
    }
```
And delete the sort from above.

Reply:
Thank you.